src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((item) => item.trim() !== '')
+    .map((item) => item.trim())
```

Comment:
You can improve the efficiency by removing the `map` call and performing the trim operation directly in the `reduce` function. This will eliminate the need for an extra iteration over the array of styles.